File: Name Programs/Fibonacci.java
class Fibonacci {
  public static void main(String[] args) {

    Scanner sc=new Scanner(System.in);
    System.out.println("Enter till which term you want fibonacci series :");
    int n=sc.nextInt();
    int firstTerm = 0, secondTerm = 1;
    System.out.println("Fibonacci Series till " + n + " terms:");

    for (int i = 1; i <= n; ++i) {
      System.out.print(firstTerm + ", ");

      // compute the next term
      int nextTerm = firstTerm + secondTerm;
      firstTerm = secondTerm;
      secondTerm = nextTerm;
    }
  }
}

File: python/fibonacci.py

def fibonacci(n):
    if n < 0:
        print("Incorrect input")
    elif n == 0:
        return 0
    elif n == 1 or n == 2:
        return 1
    else:
        return fibonacci(n-1) + fibonacci(n-2)
 

a=int(input("Enter the no of terms:"))
for i in range(0,a+1):
    print(fibonacci(i))